eq_translations/extractable_strings.py

```diff
@@ -143,6 +143,11 @@
         "description": "Answer",
         "context": "Question",
     },
+    {
+        "json_path": "$..answers[*].placeholder",
+        "description": "Placeholder text used for select field",
```

Comment:
```suggestion
        "description": "Dropdown field placeholder text",
```

Reply:
Done in [`613b25c` (#71)](https://github.com/ONSdigital/eq-translations/pull/71/commits/613b25c92c7366db1d4e3efb15ce4f32f04eed0c)